docs/ossm/injection/README.md

```diff
@@ -0,0 +1,100 @@
+## Installing the Sidecar
+### Injection
+In order to take advantage of all of Istio’s features, pods in the mesh must be running an Istio sidecar proxy.
+
+Automatic Istio sidecar injection can be enabled via labels on a namespace level or directly for a given pod. Automatic injection injects the proxy configuration at pod creation time using an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/).
+
+> **_NOTE:_** Upstream Istio offers also Manual injection using the ` istioctl kube-inject` which is not supported by OpenShift Service Mesh.
+
+After adding a correct label (see below) on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
```

Comment:
nit: idk why but correct doesn't feel right here. Maybe it implies there's only one? e.g. `the correct` instead of `a correct` would be fine?
```suggestion
After adding a valid label (see below) on a namespace, any new pods that are created in that namespace will automatically have a sidecar added to them.
```